--- grass-7.2.0/include/Make/Install.make	2017-08-24 12:27:36.000000000 -0600
+++ grass-7.2.0.mine/include/Make/Install.make	2017-08-24 13:33:14.000000000 -0600
@@ -118,6 +118,7 @@
 
 ifneq ($(findstring darwin,$(ARCH)),)
 	@# enable OSX Help Viewer
+	@# NOT INSTALLING OSX Help Viewer DOCS
 	@# /bin/ln -sfh "$(INST_DIR)/docs/html" /Library/Documentation/Help/GRASS-$(GRASS_VERSION_MAJOR).$(GRASS_VERSION_MINOR)
 endif
 
