--- grass-7.2.2/configure	2017-08-24 12:47:14.000000000 -0600
+++ grass-7.2.2.mine/configure	2017-08-24 12:48:49.000000000 -0600
@@ -1522,7 +1522,7 @@
 	*-apple-darwin*)
 	    SHLIB_CFLAGS="-fno-common"
 	    SHLIB_SUFFIX=".dylib"
-	    SHLIB_LD="cc -dynamiclib -compatibility_version \${GRASS_VERSION_MAJOR}.\${GRASS_VERSION_MINOR} -current_version \${GRASS_VERSION_MAJOR}.\${GRASS_VERSION_MINOR} -install_name \${INST_DIR}/lib/lib\${LIB_NAME}\${SHLIB_SUFFIX}"
+	    SHLIB_LD="cc -dynamiclib -compatibility_version \${GRASS_VERSION_MAJOR}.\${GRASS_VERSION_MINOR} -current_version \${GRASS_VERSION_MAJOR}.\${GRASS_VERSION_MINOR} -install_name @rpath/lib/lib\${LIB_NAME}\${SHLIB_SUFFIX}"
 	    LD_LIBRARY_PATH_VAR="DYLD_LIBRARY_PATH"
 	    ;;
 	*-sun-solaris*)
