--- grass-7.2.0/lib/python/ctypes/loader.py	2016-05-04 07:23:07.000000000 -0600
+++ grass-7.2.0.mine/lib/python/ctypes/loader.py	2017-08-23 11:37:30.000000000 -0600
@@ -125,6 +125,9 @@
         if not dyld_fallback_library_path:
             dyld_fallback_library_path = [os.path.expanduser('~/lib'),
                                           '/usr/local/lib', '/usr/lib']
+        ld_run_path = os.environ.get('LD_RUN_PATH', [])
+        for path in ld_run_path.split(os.pathsep):
+            dyld_fallback_library_path.append(path)
 
         dirs = []
 
